tests/unit_tests/dft/include/compute_tester.hpp

```diff
@@ -51,12 +51,12 @@ struct DFT_Test {
 
     enum class MemoryAccessModel { buffer, usm };
 
-    const std::vector<std::int64_t> sizes;
-    const std::vector<std::int64_t> strides_fwd;
-    const std::vector<std::int64_t> strides_bwd;
-    const std::int64_t batches;
-    const std::int64_t forward_elements;
-    const std::size_t size_total;
+    std::vector<std::int64_t> sizes;
+    std::vector<std::int64_t> strides_fwd;
+    std::vector<std::int64_t> strides_bwd;
+    std::int64_t batches;
+    std::int64_t forward_elements;
+    std::size_t size_total;
```

Comment:
Why not keeping them `const`?

Reply:
`strides_fwd` and `strides_bwd` were shadowed in `compute_inplace.hpp` so they could be modified, and this allowed me to just modify them in-place, but I decided it's not worth fixing right now so have reverted it here https://github.com/FMarno/oneMKL/pull/1/commits/d2fb2bc96d8b5fe66d4e34b744ede1ecda953710

Reply:
Oh ok, I think I remember seeing warnings because of unused `strides_fwd` and `strides_bwd` now. Do you see these warnings or are they fixed already?

Reply:
I have left the warnings as they are

Reply:
Oh, I think it would be nice to fix all the warnings we see when enabling this backend. I didn't realise this was your suggestion to fix warnings. It would make sense to revert to your first version I think!

Reply:
Ok, I've also update a small piece of code to hopefully make it clearer. https://github.com/FMarno/oneMKL/pull/1/commits/fda4b7842e0be270b8621640e11e0183a3879a47

Also you're not supposed to have const member since it can prevent them being moved, so there's another reason.